planet/src/main/webapp/places.js

```diff
@@ -0,0 +1,92 @@
+// Initializes Map, implements search box and marks locations of searches
+function initMap() {
+    var map = new google.maps.Map(document.getElementById("map"), {
```

Comment:
per https://google.github.io/styleguide/jsguide.html#features-use-const-and-let
Declare all local variables with either `const` or `let`. Do not use `var`